code/game/objects/items/bodybag.dm

```diff
@@ -334,18 +334,36 @@
 
 /obj/structure/closet/bodybag/cryobag/examine(mob/living/user)
 	. = ..()
-	if(!ishuman(bodybag_occupant))
+	var/mob/living/carbon/human/H = bodybag_occupant 
```

Comment:
Should use a proper var name instead of H

Reply:
As in like single letter var bad?